legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/finos/FinosGitlabProjectStructureExtensionProvider.java

```diff
@@ -70,6 +70,14 @@ protected int getGitLabCIFileVersion(int projectStructureVersion, int projectStr
             {
                 return 1;
             }
+            else if (projectStructureExtensionVersion == 2)
+            {
+                return 2;
+            }
+        }
+        else if(projectStructureVersion > 11)
```

Comment:
It's better not to use inequalities in this code. It will get us in trouble down the road.